pkg/operator/ceph/csi/csi.go

```diff
@@ -34,6 +34,17 @@ func ValidateAndConfigureDrivers(context *clusterd.Context, namespace, rookImage
 		v   *CephCSIVersion
 		err error
 	)
+
+	if err = SetParams(context.Clientset); err != nil {
```

Comment:
these functions should not be exported anymore?

Reply:
yes, made changes accordingly.
thanks!